pom.xml

```diff
@@ -205,7 +205,7 @@
             <dependency>
                 <groupId>javax.el</groupId>
                 <artifactId>javax.el-api</artifactId>
-                <version>3.0.0</version>
+                <version>3.0.1-b06</version>
```

Comment:
`3.0.1-b06` looks for me like Beta 6 version for release 3.0.1. Should we really use such version even it is the latest version - newer versions are available under `jakarta.el/jakarta.el-api` (see https://mvnrepository.com/artifact/jakarta.el/jakarta.el-api).

Reply:
3.0.1-b06 is rather old, but 5 years newer than 3.0.0. It looks like there never was a 3.0.1.

Newer versions would be nice, but I don't see them in the current repository. Where can 3.0.2 or 3.0.3 be downloaded?

Reply:
Group `javax.el`  got relocated to ` jakarta.el` including the artifact from ` javax.el-api` to ` jakarta.el-api`. So newer versions are found under ` jakarta.el/jakarta.el-api` like my link shows. Mvnrepository.com is a good source for finding like this (and many others).

Reply:
I suggest to handle the migration from javax.el to jakarta.el separately and either use version 3.0.1-b06 here (my preferred solution) or keep it at 3.0.0.

Reply:
@stweil I would prefer to leave this dependency at 3.0.0 in this case.

Reply:
Done. If you want, you can suggest (and commit) the desired change directly in GitHub. That might be easier.